src/Common/tests/Resources/Strings.resx

```diff
@@ -1,5 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not sure if for .resx files it was still needed. @ericstj do you know?

Reply:
Not off hand.  Test VS resx designer (for editing) and build (for reading in our code generator and compilation in resgen).